src/renderer/dx/DxRenderer.cpp

```diff
@@ -487,6 +497,19 @@ try
             THROW_HR(E_NOTIMPL);
         }
 
+#if WINVER >= _WIN32_WINNT_WINBLUE

```

Comment:
Unfortunately, we need to do these checks at runtime. We're always building for >=WINBLUE, but running on older versions. It should be easy because the `.As` will fail for the newer interface. Thanks COM! :smile: